src/Umbraco.Web/HealthCheck/Checks/Config/TrySkipIisCustomErrorsCheck.cs

```diff
@@ -38,7 +38,7 @@ public override IEnumerable<AcceptableConfiguration> Values
         {
             get
             {
-                var recommendedValue = _serverVersion >= new Version("7.5.0")
+                var recommendedValue = _serverVersion >= new Version("7.5")
```

Comment:
Alright, I see now that if you don't specify a `Build` version then it is set to -1.. you learn something new every day!

![2016-09-20_120229](https://cloud.githubusercontent.com/assets/304656/18666079/453a8348-7f2a-11e6-9b39-19c6e1e88a39.png)